<launch>
   <node name="simulation_eval" pkg="voxblox_ros" type="simulation_eval" output="screen" args="-alsologtostderr -v=3" clear_params="true">
    <param name="tsdf_voxel_size" value="1" />
    <param name="tsdf_voxels_per_side" value="64" />
    <param name="incremental" value="true" />
    <param name="add_robot_pose" value="false" />
    <param name="generate_mesh" value="true" />
    <param name="visualize" value="true" />
    <param name="visualization_slice_level" value="0.0" />
    <param name="num_viewpoints" value="50" />
    <param name="truncation_distance" value="5" />
    <param name="esdf_min_distance_m" value="10" />
    <param name="verbose" value="true" />
    <param name="max_attempts_to_generate_viewpoint" value="50" />
    <param name="world_frame" value="world" />
  </node>

</launch>
